Makefile

```diff
@@ -37,6 +37,8 @@ BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
 # Image URL to use all building/pushing image targets
 IMG ?= $(IMAGE_TAG_BASE):$(VERSION)
+PRECACHE_IMG ?= $(IMAGE_TAG_BASE)-precache:$(VERSION)
```

Comment:
Does VERSION have a default value if not specified by the caller?

Reply:
Yes, it's 0.0.3 (line 6)